src/dynamics/video_player/player/player.js

```diff
@@ -827,6 +827,8 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
                                 // in other case width will be applied twice
                                 containerStyles.width = "100%";
                             }
+                            // will shrink the page
+                            if (gallerySidebar) delete containerStyles.aspectRatio;
```

Comment:
Hi @sambua, this breaks the activeElement's aspect ratio on transition from floating to inline. 😬

Reply:
Hi @ntshcalleia how you can reproduce it? I have made some tests and this line will be only affect if it's with sidebar gallery, and no floating at all.